test/utils.js

```diff
@@ -1,23 +1,45 @@
-const mochaAsync = fn => {
+/* eslint-disable no-unused-expressions */
+import { expect } from 'chai';
+
+export const mochaAsync = fn => {
   return done => {
     fn.call().then(done, err => {
       done(err);
     });
   };
 };
 
-const removeSpace = text => {
+export const removeSpace = text => {
   return text.replace(/\s/g, '');
 };
 
-const removeTags = html => {
+export const removeTags = html => {
   return html.replace(/<\/?[^>]+(>|$)/g, '');
 };
 
-const createRandomString = () => {
+export const createRandomString = () => {
   return Math.random()
     .toString(36)
     .substring(7);
 };
 
-export { mochaAsync, removeSpace, removeTags, createRandomString };
+export const expectElementToNotExist = async (client, elementSelector) => {
+  const found = await client.isExisting(elementSelector);
+  expect(found).to.be.false;
+};
+
+export const expectEitherElementsToExist = async (client, elementSelectors) => {
```

Comment:
This is true if **any** of the elementSelectors exists. A better name would be `expectAnyElementToExist`.